plugins/woocommerce/tests/e2e-pw/tests/merchant/product-reviews.spec.js

```diff
@@ -256,7 +264,12 @@ test.describe(
 			const replyText = `Thank you for your feedback! (replied ${ Date.now() })`;
 			await replyTextArea.fill( replyText );
 
-			await page.locator( 'button.save.button.button-primary' ).click();
+			await page
+				.getByRole( 'cell', { name: 'Reply to Comment' } )
+				.getByRole( 'button', { name: 'Reply', exact: true } )
+				.click();
+
+			await expect( replyTextArea ).toBeHidden();
```

Comment:
Wait for the text area to disappear. WPCOM is a bit slower than local `wp-env` execution, and when the test in the next step goes to a given URL, the reply is not stored, and the test fails. With `toBeHidden();` we are giving some time for reply to be stored.